readme.txt

```diff
@@ -31,15 +31,15 @@ The plugin comes with several shortcodes to output jobs in various formats, and
 
 = Documentation =
 
-Documentation for the core plugin and add-ons can be found [on the docs site here](https://wpjobmanager.com/documentation/). Please take a look before requesting support because it covers all frequently asked questions!
+Documentation for the core plugin and extensions can be found [on the docs site here](https://wpjobmanager.com/documentation/). Please take a look before requesting support because it covers all frequently asked questions!
 
-= Add-ons =
+= Extensions =
 
 The core WP Job Manager plugin is free and always will be. It covers all functionality we consider 'core' to running a simple job board site.
 
-Additional, advanced functionality is available through add-ons. Not only do these extend the usefulness of the core plugin, they also help fund the development and support of core.
+Additional, advanced functionality is available through extensions. Not only do these extend the usefulness of the core plugin, they also help fund the development and support of core.
 
-You can browse available add-ons after installing the plugin by going to `Job Listings > Add-ons`. Our popular add-ons include:
+You can browse available extensions after installing the plugin by going to `WP Job Manager > Marketplace`. Our popular extensions include:
```

Comment:
One last tiny thing, this should be the main menu name: <img width="165" alt="image" src="https://github.com/Automattic/WP-Job-Manager/assets/176949/80af22b5-c947-442b-911a-2e2e4f11e2f4">

```suggestion
You can browse available extensions after installing the plugin by going to `Job Manager > Marketplace`. Our popular extensions include:
```

Reply:
Great catch! Fixed in 67a7b6f.  Thanks!